File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/MVVStationSuggestionProvider.java
package de.tum.in.tumcampusapp.component.ui.transportation;

import android.content.SearchRecentSuggestionsProvider;

public class MVVStationSuggestionProvider extends SearchRecentSuggestionsProvider {
    public final static String AUTHORITY = "de.tum.in.tumcampusapp.component.ui.transportation.MVVStationSuggestionProvider";

    public MVVStationSuggestionProvider() {
        setupSuggestions(AUTHORITY, 1);
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/MVVStationSuggestionProvider.kt
package de.tum.`in`.tumcampusapp.component.ui.transportation

import android.content.SearchRecentSuggestionsProvider

class MVVStationSuggestionProvider : SearchRecentSuggestionsProvider() {
    init {
        setupSuggestions(AUTHORITY, 1)
    }

    companion object {
        const val AUTHORITY = "de.tum.in.tumcampusapp.component.ui.transportation.MVVStationSuggestionProvider"
    }
}